### PR TITLE
Register wackymc.is-a.dev

### PR DESCRIPTION
**Pull request by Turtles127**
Register wackymc.is-a.dev with CNAME record pointing to arzejames.github.io.

### CODE REVIEW

File: domains/wacky.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Turtles127",
+           "email": "turtles127855@gmail.com",
+           "discord": "838934801972723713"
+        },
+    
+        "record": {
+            "CNAME": "arzejames.github.io"
```
Comment (CuteDog5695) [inline domains/wacky.json:9]: why did google flag your github domain as a dangerous site?

File: domains/wackymc.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Turtles127",
+           "email": "turtles127855@gmail.com",
+           "discord": "838934801972723713"
+        },
+    
+        "record": {
+            "CNAME": "arzejames.github.io"
```
Comment (CuteDog5695) [inline domains/wackymc.json:9]: why did google flag your github domain as a dangerous site?

File: domains/wackydev.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Turtles127",
+           "email": "turtles127855@gmail.com",
+           "discord": "838934801972723713"
+        },
+    
+        "record": {
+            "CNAME": "wackydev cname wackydev.netlify.app."
```
Comment (CuteDog5695) [inline domains/wackydev.json:9]:
```suggestion
            "CNAME": "wackydev.netlify.app"
```
Google also flagged this record

**Review by iostpa — CHANGES_REQUESTED** [review]
After seeing that all of the CNAME records got flagged by Google, I don't think we will merge the PR since its abuse risk

**Pull request closed without merging**